go/vt/sqlparser/ast.go

```diff
@@ -2735,6 +2736,14 @@ func (node *Show) Format(buf *TrackedBuffer) {
 		buf.Myprintf("show create trigger %v", node.Table)
 		return
 	}
+	if node.Type == "processlist" {
+		buf.Myprintf("show ")
+		if node.ShowTablesOpt != nil {
```

Comment:
rename this?

Reply:
Moved `Full` to be a bool on Show struct instead.